microprofile/7.1/25.0.0.7-beta-cl250620250508-1902-TCKResults.adoc

```diff
@@ -0,0 +1,50 @@
+
+* MicroProfile Rest Client 4.0
+
+* MicroProfile Telemetry 2.1
+
+* Jakarta EE Core Profile 10.0
+
+=== Product Name, Version and download URL (if applicable):
+
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/tck/2025-05-08_1902/openliberty-25.0.0.7-beta-cl250620250508-1902.zip[Open Liberty 25.0.0.7-beta-cl250620250508-1902]
+
+In order to certify against MicroProfile 7.1, an implementation must pass the TCK tests for the seven included MicroProfile component specifications, plus the Jakarta EE Core Profile 10.0 TCK.
+
+=== MicroProfile 7.1 Component Specification Certification Summary (Java 17)
+* xref:config/3.1/25.0.0.7-beta-cl250620250508-1902-Config-3.1-Java17-TCKResults.adoc[MicroProfile Config 3.1 TCK Result (Java 17)]
+* xref:faulttolerance/4.1/25.0.0.7-beta-cl250620250508-1902-Fault-Tolerance-4.1.1-Java17-TCKResults.adoc[MicroProfile Fault Tolerance 4.1 TCK Result (Java 17)]
+* xref:health/4.0.1/25.0.0.7-beta-cl250620250508-1902-Health-4.0.1-Java17-TCKResults.adoc[MicroProfile Health 4.0 TCK Result (Java 17)]
+* xref:jwt/2.1/25.0.0.7-beta-cl250620250508-1902-JWT-Auth-2.1-Java17-TCKResults.adoc[MicroProfile JWT Authentication 2.1 TCK Result (Java 17)]
+* xref:openapi/4.0/25.0.0.7-beta-cl250620250508-1902-Open-API-4.0.2-Java17-TCKResults.adoc[MicroProfile OpenAPI 4.0 TCK Result (Java 17)]
+* xref:restclient/4.0/25.0.0.7-beta-cl250620250508-1902-Rest-Client-4.0-Java17-TCKResults.adoc[MicroProfile Rest Client 4.0 TCK Result (Java 17)]
+* xref:telemetry/2.0/25.0.0.7-beta-cl250620250508-1902-Telemetry-2.0.1-Java17-TCKResults.adoc[MicroProfile Telemetry 2.0 TCK Result (Java 17)]
+* xref:../../jakartaee/10/coreprofile/25.0.0.7-beta-Java17-TCKResults.adoc[Jakarta EE Core Profile 10.0 TCK Result (Java 17)]
+
+=== MicroProfile 7.1 Component Specification Certification Summary (Java 21)
+* xref:config/3.1/25.0.0.7-beta-cl250620250508-1902-Config-3.1-Java21-TCKResults.adoc[MicroProfile Config 3.1 TCK Result (Java 21)]
+* xref:faulttolerance/4.1/25.0.0.7-beta-cl250620250508-1902-Fault-Tolerance-4.1.1-Java21-TCKResults.adoc[MicroProfile Fault Tolerance 4.1 TCK Result (Java 21)]
+* xref:health/4.0.1/25.0.0.7-beta-cl250620250508-1902-Health-4.0.1-Java21-TCKResults.adoc[MicroProfile Health 4.0 TCK Result (Java 21)]
+* xref:jwt/2.1/25.0.0.7-beta-cl250620250508-1902-JWT-Auth-2.1-Java21-TCKResults.adoc[MicroProfile JWT Authentication 2.1 TCK Result (Java 21)]
+* xref:openapi/4.0/25.0.0.7-beta-cl250620250508-1902-Open-API-4.0.2-Java21-TCKResults.adoc[MicroProfile OpenAPI 4.0 TCK Result (Java 21)]
+* xref:restclient/4.0/25.0.0.7-beta-cl250620250508-1902-Rest-Client-4.0-Java21-TCKResults.adoc[MicroProfile Rest Client 4.0 TCK Result (Java 21)]
+* xref:telemetry/2.0/25.0.0.7-beta-cl250620250508-1902-Telemetry-2.0.1-Java21-TCKResults.adoc[MicroProfile Telemetry 2.0 TCK Result (Java 21)]
+* xref:../../jakartaee/10/coreprofile/25.0.0.7-beta-Java21-TCKResults.adoc[Jakarta EE Core Profile 10.0 TCK Result (Java 21)]
```

Comment:
Will there be EE 11 results as well?

Reply:
We would need both for the GA but not for beta